projects/arlas-toolkit/src/lib/services/authentification/authentification.service.ts

```diff
@@ -255,6 +255,7 @@ export class AuthentificationService extends ArlasAuthentificationService {
           authConfigValue['silent_refresh_redirect_uri'] : url + 'silent-refresh.html',
         timeoutFactor: authConfigValue['timeout_factor'] !== undefined ? authConfigValue['timeout_factor'] : 0.75,
         sessionChecksEnabled: authConfigValue['session_checks_enabled'] !== undefined ? authConfigValue['session_checks_enabled'] : true,
+        sessionCheckIntervall: authConfigValue['session_checks_intervall'] !== undefined ? authConfigValue['session_checks_intervall'] : 3000,
```

Comment:
interaval

Reply:
no, the config is `intervall` with 2 L

Reply:
https://manfredsteyer.github.io/angular-oauth2-oidc/docs/classes/AuthConfig.html#sessionCheckIntervall

Reply:
my bad.